### PR TITLE
Added envoy debug log configuration

### PR DESCRIPTION
**Pull request by Amila-Rukshan**
## Purpose
Added envoy debug log configuration with example log output with its different sections.

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md [hunk 1]
```diff
@@ -113,16 +113,149 @@ Provide the log level as trailing arguments for the envoy command as follows.
...
+'date', 'Mon, 11 Apr 2022 08:18:04 GMT'
+'content-type', 'application/json'
+'access-control-allow-origin', '*'
+'access-control-allow-methods', 'GET, POST, DELETE, PUT'
+'access-control-allow-headers', 'Content-Type, api_key, Authorization'
+'server', 'envoy'
+'x-envoy-decorator-operation', 'localhost:^/v2/pet/findByStatus(\?([^/]+))?$'
+```
+
+Please follow [Command line options]({{envoy_path}}/operations/cli) for more information.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md:257]:
```suggestion
Follow [Command line options]({{envoy_path}}/operations/cli) for more information.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md [hunk 2]
```diff
@@ -113,16 +113,149 @@ Provide the log level as trailing arguments for the envoy command as follows.
...
 ```
 
+The following example configuration can log request headers and response headers, ext_authz information.
+
+```yaml
+  router:
+    environment:
+      - TRAILING_ARGS=TRAILING_ARGS=--component-log-level http:debug,http2:debug,conn_handler:debug,ext_authz:trace
+```
+Following are different sections of a example log with the above log configuration. Log generated by invoking a API request with `/pet/findByStatus?status=sold` path in [Swagger Petstore](https://petstore.swagger.io/) API deployed to Choreo Connect. Different log sections are given in the order same as the order of they are logged to the console.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md:131]:
```suggestion
The following are different sections of an example log with the above log configuration. This log is generated by invoking an API request with `/pet/findByStatus?status=sold` path in [Swagger Petstore](https://petstore.swagger.io/) API deployed to Choreo Connect. Different log sections are given in the order same as the order in which they are logged to the console.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md [hunk 3]
```diff
@@ -113,16 +113,149 @@ Provide the log level as trailing arguments for the envoy command as follows.
...
-For example, Add following line to the docker-compose.yaml in the directory `<CHOREO-CONNECT_HOME>/docker-compose/<choreo-connect>/`.
+For example, add the following line to the docker-compose.yaml file in the directory `<CHOREO-CONNECT_HOME>/docker-compose/<choreo-connect>/`.
 
 ```yaml
   router:
     environment:
       - TRAILING_ARGS=--log-level trace
 ```
 
+The following example configuration can log request headers and response headers, ext_authz information.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md:124]:
```suggestion
The following example configuration can log request headers and response headers, and ext_authz information.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md [hunk 4]
```diff
@@ -113,16 +113,149 @@ Provide the log level as trailing arguments for the envoy command as follows.
...
+'date', 'Mon, 11 Apr 2022 08:18:04 GMT'
+'content-type', 'application/json'
+'access-control-allow-origin', '*'
+'access-control-allow-methods', 'GET, POST, DELETE, PUT'
+'access-control-allow-headers', 'Content-Type, api_key, Authorization'
+'server', 'envoy'
+'x-envoy-decorator-operation', 'localhost:^/v2/pet/findByStatus(\?([^/]+))?$'
+```
+
+Follow [Command line options]({{envoy_path}}/operations/cli) for more information.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md:257]: Not sure what this link is. @pubudu538 is this how this is supposed to be?

Comment (pubudu538) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/adding-debug-logs.md:257]: Choreo Connect is built on top of Envoy. That is the reason for having this link.

**Review by samgnaniah — APPROVED** [review]

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**